lib/node.js

```diff
@@ -468,7 +493,25 @@ declare module "punycode" {
 }
 
 declare module "querystring" {
-  // TODO
+  declare function stringify(
+    obj: Object,
+    separator?: string,
+    equal?: string,
+    options?: {
+      encodeURIComponent?: (str: string) => string;
+    }
+  ): string;
+  declare function parse(
+    str: string,
+    separator?: string,
+    equal?: string,
+    options?: {
+      decodeURIComponent?: (str: string) => string;
+      maxKeys?: number;
+    }
+  ): any;
+  declare function escape(str: string): string;
+  declare function unescape(str: string, decodeSpaces?: boolean): string;
```

Comment:
It's just poorly documented :disappointed: 

https://github.com/joyent/node/blob/v0.12.2/lib/querystring.js#L108
https://github.com/joyent/node/blob/v0.10.38/lib/querystring.js#L107
https://github.com/iojs/io.js/blob/v1.6.3/lib/querystring.js#L80